apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -126,6 +126,12 @@ export type AttestationServiceConfiguration = t.TypeOf<
 const AzureConfig = t.type({
   appInsights: AzureAppInsightsConfig,
   cosmos: AzureCosmosConfig,
+  queue: t.type({
```

Comment:
this `queue` property should be removed and the properties have to be in `storage.walletInstances.queues`